rust/index/bindings.cpp

```diff
@@ -193,76 +195,208 @@ class Index
     {
         if (!index_inited)
         {
-            std::runtime_error("Index not inited");
+            throw std::runtime_error("Index not inited");
         }
         appr_alg->resizeIndex(new_size);
     }
 };
 
+// All these methods except for len and capacity can throw std::exception
```

Comment:
To be clear, the method themselves don't actually `throw` right? Just that something we call inside them can throw?

Reply:
Yes, will clean up descriptions and include a comment of how we are handling exceptions across FFI

Reply:
Added